helm_deploy/values-preprod.yaml

```diff
@@ -24,3 +24,4 @@ generic-service:
     CONFIRM_COURT_RETURN_ENABLED: 'false'
     SUPPORTING_MULTITRANSACTIONS_ENABLED: 'true'
     EXPORT_SPREADSHEET_ID: "15tjbOMJUAGSVQ4tpcEi83jWkzkMNrptUH6SC0ZHslqY"
+    SERVICE_IS_UNAVAILABLE: "true"
```

Comment:
Just checking - assume once this has been tested this will revert back to false?

Reply:
Also did you try adding the test for the environment variable and have the same issues as in BVL?

Reply:
yes, will change true to false after checking
No, didn't try the test. Didn't think have enough time especially as were testing it manually